app/views/meals/index.html.erb

```diff
@@ -1,2 +1,91 @@
-<h1>Meals#index</h1>
-<p>Find me in app/views/meals/index.html.erb</p>
+<div class="top-nav">
+  <div class="box index-container">
+    <div class="social">
+      <p>SHARE A MEAL, SHARE A CULTURE. get a taste of homecooked blachblahblah</p>
+    </div>
+    <div>
+      <ul class="social">
+      <li><i class="fa-brands fa-facebook-f"></i></li>
+      <li><i class="fa-brands fa-instagram"></i></li>
+      <li><i class="fa-brands fa-facebook-f"></i></li>
+      <li><i class="fa-brands fa-facebook-f"></i></li>
+      </ul>
+    </div>
+  </div>
+</div>
+
+<%# main contain after pink bar %>
+
+<div class="index-container">
+<%# Navigation bar %>
+<div class="topnav">
+  <div><button class="fa-regular fa-circle-user fa-2x collapsible" type="button" data-bs-toggle="collapse" data-bs-target="#demo" aria-expanded="false" aria-controls="collapseExample">
+  </button></div>
+   <div class="title"><img src="https://res.cloudinary.com/ddaoe4tdm/image/upload/v1668531915/tastehome2_msqitj.png" width="180px"></div>
+<div class="cart"><i class="fa-solid fa-bag-shopping fa-2x"></i></div>
+
+</div>
+
+</div>
+
+<div id="demo" class="collapse dropdown-content">
+<ul class="nav-content">
+<li class="nav-menu"><a href="<%= new_user_session_path %>">------ Sign in</a></li>
+<li class="nav-menu"><a href="https://minimalistbaker.com/about/">------ Create an account</a></li>
+<li class="nav-menu"><a href="https://minimalistbaker.com/about/">------ Offer a meal</a></li>
+<li class="nav-menu"><a href="https://minimalistbaker.com/about/">------ Get a meal</a></li>
+<li class="nav-menu"><a href="https://minimalistbaker.com/about/">------ View orders</a></li>
+<li class="nav-menu"><%= link_to "------ Logout", destroy_user_session_path, data:{turbo_method: :delete}, class: "dropdown-item" %></li>
```

Comment:
Logout link is indented while other links are not - try to remove the dropdown-item class?